index.js

```diff
@@ -289,4 +285,24 @@ async function handlePun(message) {
 		message.channel.send('The joke is that this API doesn\'t work.');
 		console.log(error);
 	});
+
+	async function YOLOStatus(CompanyName) {
+		const options = {
+			method: 'GET',
+			url: 'https://apidojo-yahoo-finance-v1.p.rapidapi.com/auto-complete',
+			params: {q: CompanyName, region: 'US'},
+			headers: {
+			  'x-rapidapi-key': 'SIGN-UP-FOR-KEY',
```

Comment:
This definitely won't work. Seems to me like this API requires a key to be defined. You would define this in the the .env file and reference it here (like in the weather API).